rm2/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "rm2",
+  "version": "1.0.0",
+  "description": "building react app with webpack from scratch",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start:dev": "cross-env NODE_ENV=development node_modules/.bin/webpack-dev-server --watch",
+    "start:prod": "cross-env NODE_ENV=production node_modules/.bin/webpack-dev-server -p",
+    "build:dev": "cross-env NODE_ENV=development node_modules/.bin/webpack-dev-server --watch",
+    "build:prod": "cross-env NODE_ENV=production node_modules/.bin/webpack-dev-server -p"
```

Comment:
Let me know when you're done with the ESLint & Prettier scripts.

Reply:
I have added that but not sure about the prettier fix

Reply:
would need more clarity on that

Reply:
You're almost done here!
It's just the "--write" flag that defines whether it's a check or a fix.
https://prettier.io/docs/en/cli.html

<img width="559" alt="Screenshot 2022-04-27 at 16 27 56" src="https://user-images.githubusercontent.com/25229823/165554974-d5bb8a76-087c-4501-9521-432cbdf777bb.png">

Reply:
Thanks